pkg/ottl/boolean_value.go

```diff
@@ -181,3 +191,25 @@ func (p *Parser[K]) newBooleanValueEvaluator(value *booleanValue) (BoolExpr[K],
 	}
 	return boolExpr, nil
 }
+
+func (p *Parser[K]) newConverterEvaluator(val *booleanValue) (BoolExpr[K], error) {
+	getter, err := p.newGetter(value{
```

Comment:
It feels a little funny to have the parsing here depend on the parsing of a separate AST structure. I think it would be better if we split out the Converter handling in `newGetter` into a separate method and called that here. What do you think?

Reply:
Sure I can make that change.  It'll save us from having to create a new `value` struct instance.

Reply:
@evan-bradley done.